src/statsig/index.test.ts

```diff
@@ -106,20 +106,16 @@ describe('Statsig helpers', () => {
 
   describe('getFeatureGate', () => {
     it('returns default values if getting statsig feature gate throws error', () => {
-      mocked(Statsig.checkGate).mockImplementation(() => {
```

Comment:
I thought `mocked(...)` was the preferred way of doing this

Reply:
weird, i think this was something odd going on with automerge